pages/tutorials/parallel_builds.md

```diff
@@ -35,6 +35,30 @@ buildkite-agent start --tags queue=test
 buildkite-agent start --tags queue=deploy
 ```
 
+#### Coordinating multiple agents
+
+>🛠️ Experimental feature
+> To use it, set <code>experiment="agent-api"</code> in your <a href="/docs/agent/v3/configuration#experiment"> agent configuration</a>.
+> This also requires Agent v3.47.0 or later.
+
+Multiple agents on a single host can sometimes interfere with one another. For example, a pipeline might contain commands like `docker prune` or `apt upgrade`, but these commands fail if another job is running the same command at the same time.
```

Comment:
General tightening:

```suggestion
Multiple agents on a single host can sometimes interfere with one another. For example, a pipeline might contain commands like `docker prune` or `apt upgrade`, but these commands fail if another job runs the same command at the same time.
```

You could also consider changing "at the same time" to "simultaneously," but I'm not sure it makes it any clearer.